internal/js/modules/k6/browser/common/locator.go

```diff
@@ -325,6 +325,12 @@ func (l *Locator) fill(value string, opts *FrameFillOptions) error {
 	return l.frame.fill(l.selector, value, opts)
 }
 
+// First will return the first child of the element matching the locator's
+// selector.
+func (l *Locator) First() *Locator {
+	return NewLocator(l.ctx, fmt.Sprintf("%s >> nth=0", l.selector), l.frame, l.log)
```

Comment:
I would usually not comment on such a thing, but since we want to optimize the module, I'd use an old-fashioned string concat here, instead of the less-performant `Sprintf`, which may allocate and run a lot of reflection code:

```suggestion
	return NewLocator(l.ctx, l.selector + " >> nth=0", l.frame, l.log)
```